checkQC/handlers/q30_handler.py

```diff
@@ -32,37 +32,39 @@ def check_qc(self):
             lane_nbr = int(error_dict["lane"])
             read = error_dict["read"]
             percent_q30 = error_dict["percent_q30"]
-            is_index_read = error_dict["percent_q30"]
-            read_or_index_text = "index read" if is_index_read else "read"
+            index = error_dict["index"]
+
             
             if self.error() != self.UNKNOWN and percent_q30 < self.error():
                 yield QCErrorFatal(
-                    "%Q30 {:.2f} was too low on lane: {} for {}: {}".format(
+                    "%Q30 {:.2f} was too low on lane: {} for read: {}, index: {}".format(
                         percent_q30,
                         lane_nbr,
-                        read_or_index_text,
-                        read
+                        read,
+                        index
                     ),
                     ordering=lane_nbr,
                     data={
                         "lane": lane_nbr, 
                         "read": read,
+                        "index": index,
                         "percent_q30": percent_q30, 
                         "threshold": self.error()
                     }
                 )
             elif self.warning() != self.UNKNOWN and percent_q30 < self.warning():
                 yield QCErrorWarning(
-                    "%Q30 {:.2f} was too low on lane: {} for {}: {}".format(
+                    "%Q30 {:.2f} was too low on lane: {} for read: {}, index: {}".format(
```

Comment:
Ah sorry if I was unclear, what I meant was: 
For the first non-index read the message should be: 
`%Q30 {:.2f} was too low on lane: {} for read: 1`

For the first index read it should be: 
`%Q30 {:.2f} was too low on lane: {} for index read: 1`

For the second index read (if it exists) it should be: 
`%Q30 {:.2f} was too low on lane: {} for index read: 2`

For the second non-index read (if it exists) it should be: 
`%Q30 {:.2f} was too low on lane: {} for read: 2`

Reply:
So you previous implementation was kind of correct but the index (that is, the "number ID" not the index read, this word is so confusing in this context :facepalm: ) of the reads need to be corrected. 

It makes no sense to talk about index read 3, when there are only 2 index reads. 

Let me know if this is unclear.

Reply:
Thank you for the clarification, I have a question on where to use the number ID...should it be like this now?

For the first non-index read the message should be:
`%Q30 {:.2f} was too low on lane: {} for read: 1`

For the first index read it should be:
`%Q30 {:.2f} was too low on lane: {} for number ID: 1`

For the second index read (if it exists) it should be:
`%Q30 {:.2f} was too low on lane: {} for number ID: 2`

For the second non-index read (if it exists) it should be:
`%Q30 {:.2f} was too low on lane: {} for read: 2`

Reply:
Ah no, I didn't mean it literally, just tried to say index without mixing it up index reads. This is still accurate: https://github.com/Molmed/checkQC/pull/113#discussion_r1551696310